services/motion/motion.go

```diff
@@ -188,6 +188,14 @@ type Service interface {
 		slamName resource.Name,
 		extra map[string]interface{},
 	) (bool, error)
+	MoveOnMapNew(
```

Comment:
I recommend making this a `motion.MoveOnMapReq` struct to contain all parameters other than `ctx`, similar to how `MoveOnGlobeNew` is